docassemble/AssemblyLine/al_general.py

```diff
@@ -1031,6 +1038,160 @@ def address_block(
             )
         )
 
+    def pronoun(self, **kwargs):
+        """Returns a pronoun as appropriate, based on attributes.
+
+        The pronoun could be "you," "her," "him," "it," or "them". It depends
+        on the `gender` and `person_type` attributes and whether the individual
+        is the current user.
+
+        If the user selected specific pronouns, they take priority over
+        gender (only if they chose a pronoun from the list)
+
+        Args:
+            **kwargs: Additional keyword arguments.
+
+        Returns:
+            str: The appropriate pronoun.
+        """
+        if self == this_thread.global_vars.user:
+            output = word("you", **kwargs)
+        elif (
+            hasattr(self, "pronouns")
+            and isinstance(self.pronouns, DADict)
+            and len(self.pronouns.true_values()) == 1
+            and self.pronouns.true_values()[0]
+            in ["she/her/hers", "he/him/his", "they/them/theirs", "ze/zir/zirs"]
```

Comment:
This is a super great structure IMO, but I think a really good improvement would be to accept any pronouns that have two slashes in them. `he/him/his` is pretty standard (I have seen slashes for people who are okay with multiple sets, like (she/they), but it's usually only two), and it doesn't feel great to have the option for users to enter their own pronouns, but then to ignore them.

Reply:
You're right, I was just getting lazy. I'll add that before merging.